app/react/Layout/DocumentsList.js

```diff
@@ -158,7 +158,7 @@ class DocumentsList extends Component {
                     clickOnDocument: this.clickOnDocument,
                     onSnippetClick: this.props.onSnippetClick,
                     deleteConnection: this.props.deleteConnection,
-                    onEndScroll: this.onEndScroll,
+                    onEndScroll: this.loadNextGroupOfEntities,
```

Comment:
I like this semantics much better!

Reply:
Still, the CollectionViewer "interface" needs its onEndScroll prop renamed to loadNextGroupOfEntities. Sorry it's not pushed, we discussed this just really late yesterday.